spec/services/change_course_spec.rb

```diff
@@ -100,10 +100,63 @@
     end
 
     describe 'emails' do
-      it 'sends an email' do
+      it 'sends an email when the course changes' do
+        old_course_option = application_choice.course_option
+        new_course = create(:course, provider: old_course_option.course.provider)
+        new_course_option = create(:course_option, course: new_course)
+
+        change_course = described_class.new(
+          actor: provider_user,
+          application_choice:,
+          course_option: new_course_option,
+          update_interviews_provider_service:,
+        )
+
         change_course.save!
 
-        expect(CandidateMailer).to have_received(:change_course).with(application_choice, application_choice.original_course_option)
+        expect(CandidateMailer).to have_received(:change_course).with(application_choice, old_course_option)
+      end
+
+      it 'does not send an email when only the site changes and placement is not auto-selected' do
```

Comment:
And I think it would be good to test the opposite:
`it 'sends an email when only the site changes and the placement is not auto-selected (the candidate selected it themselves)'.`